imagebuildah/stage_executor.go

```diff
@@ -373,10 +373,7 @@ func (s *StageExecutor) Copy(excludes []string, copies ...imagebuilder.Copy) err
 			return errors.New("COPY --parents is not supported")
 		}
 		if len(cp.Excludes) > 0 {
-			if cp.Download {
-				return errors.New("ADD --excludes is not supported")
-			}
-			return errors.New("COPY --excludes is not supported")
+			excludes = append(excludes, cp.Excludes...)
```

Comment:
Append to `slices.Clone(excludes)`, please.  We don't own `excludes` and don't want to inadvertently modify the slice that was passed to us.